en/docs/administer/production-deployment-guidelines.md

```diff
@@ -0,0 +1,701 @@
+# Production Deployment Guidelines
```

Comment:
I think we removed this intentionally, check with @somindatommy

Reply:
Yes, we removed this. Some content is this doc is already in other docs. And the content that are not in other docs was moved to new docs.

Reply:
removed from doc space